src/main/java/uk/gov/hmcts/ccd/data/draft/DefaultDraftRepository.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.ccd.data.draft;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.ccd.ApplicationParams;
+import uk.gov.hmcts.ccd.data.SecurityUtils;
+import uk.gov.hmcts.ccd.domain.model.draft.CreateCaseDataContentDraft;
+import uk.gov.hmcts.ccd.domain.model.draft.Draft;
+import uk.gov.hmcts.ccd.endpoint.exceptions.ServiceException;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+
+@Named
+@Qualifier(DefaultDraftRepository.QUALIFIER)
+@Singleton
+public class DefaultDraftRepository implements DraftRepository {
+
+    private static final Logger LOG = LoggerFactory.getLogger(DefaultDraftRepository.class);
+    public static final String QUALIFIER = "default";
+    private static final String DRAFT_ENCRYPTION_KEY_HEADER = "Secret";
+
+    private final RestTemplate restTemplate;
+    private final SecurityUtils securityUtils;
+    private final ApplicationParams applicationParams;
+
+    @Inject
+    public DefaultDraftRepository(
+            final RestTemplate restTemplate,
+            final SecurityUtils securityUtils,
+            final ApplicationParams applicationParams) {
+        this.restTemplate = restTemplate;
+        this.securityUtils = securityUtils;
+        this.applicationParams = applicationParams;
+    }
+
+    @Override
+    public Draft set(final CreateCaseDataContentDraft draft) {
```

Comment:
The name of the method is too generic I believe. This implementation is exclusively to create new drafts, not updating existing as `set` may suggest.

Reply:
This is indeed changed and called `save` in the follow up PR for 2362:
https://github.com/hmcts/ccd-data-store-api/pull/115